OrbitQt/ConnectToStadiaWidget.cpp

```diff
@@ -0,0 +1,378 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ConnectToStadiaWidget.h"
+
+#include <QMessageBox>
+#include <QRadioButton>
+#include <QSettings>
+#include <optional>
+#include <utility>
+
+#include "Error.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/Result.h"
+#include "OrbitGgp/Client.h"
+#include "OverlayWidget.h"
+#include "servicedeploymanager.h"
+
+namespace {
+const QString kRememberChosenInstance{"RememberChosenInstance"};
+}  // namespace
+
+namespace orbit_qt {
+
+using OrbitGgp::Instance;
+using OrbitSshQt::ScopedConnection;
+
+ConnectToStadiaWidget::ConnectToStadiaWidget(QWidget* parent)
+    : QWidget(parent), ui_(std::make_unique<Ui::ConnectToStadiaWidget>()) {
+  ui_->setupUi(this);
+
+  parent->installEventFilter(this);
```

Comment:
I did revise this part and its now done differently. The main challenge I had was figuring out the best way of grouping everything in this widget besides the radioButton, so it can be enabled separately. This was problem because the radioButton and the rest of the content are not distinct rectangles, but overlapping. 
I tried QStackedWidget with QStackedLayout, but that sadly did not lead anywhere, because even when all stacks in a stackedlayout are visible at the same time, they are not transparent for mouseevents, so I would needed to have filtered and reapplied the these.

How I did it now is that everything is grouped in a QFrame called contentFrame and after the ui_->setupUi(this) call, I remove the radioButton from contentFrame and place it on top of it at the same position. ptal